apps/indexers/nft-marketplace/src/logger.ts

```diff
@@ -0,0 +1,16 @@
+import { createLogger, format, transports } from 'winston';
+import chalk from 'chalk';
+
+const { combine, printf } = format;
+
+const actionFormat = printf(({ message, label }) => `${chalk.magenta('[')}${label}${chalk.magenta(']')} ${message}`);
+
+const logger = createLogger({
```

Comment:
Probably you know, but there is a [built-in structured logger](https://github.com/subsquid/squid/tree/master/util/logger) available in mappings via `ctx.log`.

Reply:
Yeah, this part has been removed